Quaver.Shared/Screens/Select/UI/Modifiers/DrawableModifierBool.cs

```diff
@@ -31,7 +32,15 @@ public override List<DrawableModifierOption> CreateModsDialogOptions()
                 new DrawableModifierOption(this, "ON", (o, e) =>
```

Comment:
It's worth mentioning that when you click the on/off button, the click event fires again. This should preferably be prevented here as well if we're going this route.